owl-bot-post-processor/main.sh

```diff
@@ -30,6 +33,42 @@ copy_one_api() {
   STAGING_DIR=owl-bot-staging/$1
   PACKAGE_DIR=apis/$1
 
+  # We don't expect googleapis-gen to contain the right (or potentially
```

Comment:
Just a question: Are we sure that, whatever had happened in googleapis-gen, we still get to execute this? Or could that fail in such a way that this is not called?

Reply:
This function is only called if there's something in the owl-bot-staging directory for an API, which suggests it's managed to copy the API over from googleapis-gen already. If an API fails to generate in googleapis-gen itself, that's a whole other issue, and one which I assume Bazel Bot handles.